nominal/cli/dataset.py

```diff
@@ -64,3 +67,45 @@ def get(rid: str, base_url: str, token: str | None) -> None:
     client = get_client(base_url, token)
     dataset = client.get_dataset(rid)
     print(dataset)
+
+
+@dataset_cmd.command("summarize")
+@click.option("-r", "--rid", required=True, multiple=True, help="RID(s) of the dataset(s) to summarize")
+@click.option(
+    "-c",
+    "--csv",
+    type=click.Path(exists=False, dir_okay=False, resolve_path=True, path_type=pathlib.Path),
+    help="If provided, a path to write the description to as a CSV",
+)
+@click.option("--show-rids", is_flag=True, help="If provided, show channel / dataset RIDs as part of tabulated output")
```

Comment:
with click flags, generally prefer using `--show-rids/--no-show-rids` - click will auto-detect that it's a flag, and gives you a way both to explicitly enable and disable and you can still have either as the default

but I'd also just always include this